pilz_robot_programming/test/integrationtests/tst_sequence_with_gripper.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+# Copyright (c) 2019 Pilz GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+from rospkg import RosPack
+import numpy
+
+from geometry_msgs.msg import Point
+
+from pilz_robot_programming.robot import *
+from pilz_robot_programming.commands import *
+
+from pilz_industrial_motion_testutils.integration_test_utils import *
+from pilz_industrial_motion_testutils.robot_motion_observer import RobotMotionObserver
+
+API_VERSION = "1"
+
+class TestSequenceWithGripper(unittest.TestCase):
+
+    _GRIPPER_GROUP_NAME = "gripper"
+    _GRIPPER_POSE_CLOSED = 0.001
+
+    _TOLERANCE_POSITION_COMPARE = 1e-3
+
+    def setUp(self):
+        if not 'ros_init_done' in globals():
+            rospy.init_node('tst_sequence_with_gripper')
+            global ros_init_done
+
+        self.robot = Robot(API_VERSION)
+        self.robot_motion_observer = RobotMotionObserver(group_name=self._GRIPPER_GROUP_NAME)
```

Comment:
this observer is unused. the respective import above can be removed, too.

Reply:
Removed